### PR TITLE
Feature/ui no initial sort

### PR DESCRIPTION
**Pull request by astewes**
-  Adds the ability to not sort by default (but then allow sorting based on each table heading)

### CODE REVIEW

**Comment by gaarf** [comment]
LGTM

File: modules/cask-angular-sortable/sortable.js [hunk 1]
```diff
@@ -40,7 +40,10 @@ function caskSortableDirective ($log) {
 
       var headers = element.find('th'),
           defaultPredicate,
-          defaultReverse;
+          defaultReverse,
+          noInitialSort;
+
+      noInitialSort = (attrs.noInitialSort === 'true'? true: false);
```
Comment (elfenheart) [inline modules/cask-angular-sortable/sortable.js:46]: is the value always get passed as string?

**Comment by astewes** [comment]
@elfenheart correct, the no-initial-sort attribute is intended to be used in template files with a "true" or "false" value

File: modules/cask-angular-sortable/sortable.js (hunk 1 above)
Comment (ajainarayanan) [inline modules/cask-angular-sortable/sortable.js:46]: The default value would be false which means `noInitialSort` is not enabled by default (which is the existing behavior)

**Comment by ajainarayanan** [comment]
:+1: LGTM

**Pull request merged**